### ISSUE TITLE
EC drain for command pods too high

### ISSUE THREAD

**Issue by theJesuit** [steamp0rt/Kerbalism#64]
It seems that whatever the planner is calling "communications (control)" is being drained constantly.

In the last version I was using (## v1.6.0 for KSP 1.4.3 - 2018-05-26) that same requirement did not seem to drain EC, but was 12units/sec in the planner. 

Now in the latest release off Github (downloaded 2018-06-18 it drains fast, needing four OX-STATS facing the sun to keep up with EC drain PER command pod and probe.

**Comment by PiezPiedPy** [comment]
the internal transmitters are now constantly active, the EC is usually about 0.02 EC/s and is shown in the planner as "communications (control)", they only transmit telemetry and control and do not transmit science.
can you send a screenshot of the values in the planner and your output log.

**Comment by PiezPiedPy** [comment]
![screenshot2](https://user-images.githubusercontent.com/17555434/41530374-f9769038-72e7-11e8-9774-7d692eda174d.png)

**Comment by PiezPiedPy** [comment]
Also see Issue #53

**Comment by theJesuit** [comment]
I feel like a dunce now.  I went back to replace this was all sorted for a default profile (as I had checked it before, and released that in a separate cfg I had made the adjustments for the 

@PART[*]:HAS[@MODULE[ModuleDataTransmitter]]:NEEDS[FeatureScience]
	{	@MODULE[ModuleDataTransmitter]
			{	**@packetInterval *= 0.01**
			}
	}

this being the culprit.  This was for me nerfing (OP?) the data rates for transmitting science.

I hadn't adjusted packetResourceCost as well.  When I did it resolved the issue.  My apologies!
